src/map.rs

```diff
@@ -432,6 +432,31 @@ fn example() -> Result<(), Box<dyn std::error::Error>> {
     pub fn into_fst(self) -> raw::Fst<D> {
         self.0
     }
+
+    /// Maps the underlying data of the fst Map to another data type.
+    ///
+    /// # Example
+    ///
+    /// This example shows that you can map an fst Map based on a `Vec<u8>`
+    /// into an fst Map based on a `Cow<[u8]>`, it can also work with a
+    /// reference counted type (e.g. `Arc`, `Rc`)
+    ///
+    /// ```rust
```

Comment:
I don't think you need `rust` here.

Reply:
Yeah, I just copy/pasted the example above ;)

Reply:
Ah yeah, `fst` is a very old crate. Maybe it was needed at one point? Or I just slipped. It happens. :P